### ISSUE TITLE
Dish Anywhere Audio Support

### ISSUE THREAD

**Issue by richardfrost** [richardfrost/AdvancedProfanityFilter#156]
#### :movie_camera: The Site
www.dishanywhere.com

#### :speech_balloon: Subtitle Element/Node
```html
<span id="bmpui-id-286" class="bmpui-ui-subtitle-label" style="left: 0%; top: 93.3333%; font-size: 28.2667px; letter-spacing: 4.29px;">
  AND YOU HUG YOURSELF
</span>
```

**Comment by richardfrost** [comment]
You'll need to use a custom dev build to test right now.

[extension-chrome.zip](https://github.com/richardfrost/AdvancedProfanityFilter/files/3483498/extension-chrome.zip)
[extension-firefox.zip](https://github.com/richardfrost/AdvancedProfanityFilter/files/3483499/extension-firefox.zip)
 
Add the following [Custom Audio Muting Site](https://github.com/richardfrost/AdvancedProfanityFilter/wiki/Custom-Audio-Muting-Site): 

```json
{
  "customAudioSites": {
    "www.dishanywhere.com": {
      "className": "bmpui-ui-subtitle-label",
      "tagName": "SPAN"
    }
  }
}
```

**Comment by richardfrost** [comment]
A beta version was released, which you can test here (#159). Once this has been validated and officially released this issue will be closed.

**Comment by richardfrost** [comment]
Version 2.2.0 has been released and should support dish anywhere by default.
